src/status_im/ethereum/transactions/core.cljs

```diff
@@ -2,14 +2,27 @@
   (:require [re-frame.core :as re-frame]
             [status-im.constants :as constants]
             [status-im.ethereum.decode :as decode]
+            [status-im.ethereum.encode :as encode]
             [status-im.ethereum.json-rpc :as json-rpc]
-            [status-im.ethereum.transactions.etherscan :as transactions.etherscan]
+            [status-im.ethereum.core :as ethereum]
+            [status-im.ethereum.tokens :as tokens]
             [status-im.utils.fx :as fx]
             [status-im.utils.money :as money]
             [status-im.wallet.core :as wallet]))
 
 (def confirmations-count-threshold 12)
 
+(def etherscan-supported? #{:testnet :mainnet :rinkeby})
+
+(let [network->subdomain {:testnet "ropsten" :rinkeby "rinkeby"}]
+  (defn get-transaction-details-url [chain hash]
+    {:pre [(keyword? chain) (string? hash)]
+     :post [(or (nil? %) (string? %))]}
+    (when (etherscan-supported? chain)
+      (let [network-subdomain (when-let [subdomain (network->subdomain chain)]
+                                (str subdomain "."))]
+        (str "https://" network-subdomain "etherscan.io/tx/" hash)))))
```

Comment:
we still need etherescan? we can't abandon it?

Reply:
this is just for showing the transaction details URL if someone wants to check it out in the browser, it's not etherscan api

Reply:
right! thanks